tensorboard/meta.yaml

```diff
@@ -1,31 +1,30 @@
 {% set name = "tensorboard" %}
-{% set version = "2.6.0" %}
+{% set version = "2.8.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/py3/t/tensorboard/tensorboard-{{ version }}-py3-none-any.whl
-  sha256: f7dac4cdfb52d14c9e3f74585ce2aaf8e6203620a864e51faf84988b09f7bbdb
+  sha256: 65a338e4424e9079f2604923bdbe301792adce2ace1be68da6b3ddf005170def
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  skip: True  # [py<37]
   script:
     - python -m pip install --no-deps --ignore-installed ./tensorboard-{{ version }}-py3-none-any.whl
   entry_points:
     - tensorboard = tensorboard.main:run_main
 
 requirements:
   host:
-    - python >=3.2
+    - python
     - pip
   run:
-    - python >=3.2
+    - python
     - wheel >=0.26
     - numpy >=1.12.0
-    - six >=1.10.0
     - protobuf >=3.6.0
     - werkzeug >=0.11.15
     - markdown >=2.6.8
```

Comment:
google-auth-oauthlib  should be `>= 0.4.1,< 0.5`, see https://github.com/tensorflow/tensorboard/blob/ce02d854bc41f46d3e2d7069cd5846b3126d481f/tensorboard/pip_package/requirements.txt#L24

Reply:
`requests >= 2.21.0,<3` is required, see https://github.com/tensorflow/tensorboard/blob/2.8.0/tensorboard/pip_package/requirements.txt#L24

Reply:
`setuptools >=41.0.0` is required, see https://github.com/tensorflow/tensorboard/blob/ce02d854bc41f46d3e2d7069cd5846b3126d481f/tensorboard/pip_package/requirements.txt#L29